manifests/xbuild.json

```diff
@@ -0,0 +1,44 @@
+{
+  "rust_crate": "xbuild",
+  "template": null,
+  "latest": {
+    "version": "0.2.0"
+  },
+  "0.2": {
+    "version": "0.2.0"
+  },
+  "0.2.0": {
+    "x86_64_linux_gnu": {
+      "url": "https://github.com/rust-mobile/xbuild/releases/download/v0.2.0/xbuild-linux-x64",
+      "checksum": "9c294809ec3cb314b34e9f644b5bbf6ed262c25c1eeb78a3691836a95bc58e0d",
+      "bin": "xbuild-linux-x64"
+    },
+    "x86_64_macos": {
+      "url": "https://github.com/rust-mobile/xbuild/releases/download/v0.2.0/xbuild-macos-x64",
+      "checksum": "e357205fde5262d77b8b1dc1cb066a4f7b4edd0a83b71b3513ae466b08343305",
+      "bin": "xbuild-macos-x64"
+    },
+    "x86_64_windows": {
+      "url": "https://github.com/rust-mobile/xbuild/releases/download/v0.2.0/xbuild-windows-x64.exe",
+      "checksum": "ce503cd64e6449d1c3e3bc2bfb621e9dd3ebfc775e99c97f98c2ab145e533946",
+      "bin": "xbuild-windows-x64.exe"
+    }
+  },
+  "0.1.0+4": {
+    "x86_64_linux_gnu": {
+      "url": "https://github.com/rust-mobile/xbuild/releases/download/v0.1.0%2B4/xbuild-linux-x64",
+      "checksum": "5da29e47e4227ca0c6af335b80e20c9b0ae673ac3cfdcf235a4e2f5a9b57796f",
+      "bin": "xbuild-linux-x64"
+    },
+    "x86_64_macos": {
+      "url": "https://github.com/rust-mobile/xbuild/releases/download/v0.1.0%2B4/xbuild-macos-x64",
+      "checksum": "846c704d512912a55dbe1fdff8d50d6408a17a557931ed6d5fa070fc24fbf370",
+      "bin": "xbuild-macos-x64"
+    },
+    "x86_64_windows": {
+      "url": "https://github.com/rust-mobile/xbuild/releases/download/v0.1.0%2B4/xbuild-windows-x64.exe",
+      "checksum": "e24df52574fbba3b707ec85502f7515a78401cf06bae322204def565f62f555c",
+      "bin": "xbuild-windows-x64.exe"
+    }
```

Comment:
(version_range doesn't affect already generated manifests. -- It is a design to handle the case where the URL has changed.)

```suggestion
```